GitUI/TranslatedStrings.cs

```diff
@@ -312,6 +312,17 @@ public static void Reinitialize()
         private readonly TranslationString _scriptErrorOptionWithoutRevisionText = new("A valid revision is required to substitute the argument options");
         private readonly TranslationString _scriptText = new("Script");
 
+        private readonly TranslationString _seeErrorMessage = new("See error message");
+        private readonly TranslationString _hideErrorMessage = new("Hide error message");
+        private readonly TranslationString _reportTheIssue = new("Report the issue");
+        private readonly TranslationString _relaunchGitExtensions = new("Relaunch Git Extensions");
```

Comment:
```suggestion
        private readonly TranslationString _relaunchGitExtensions = new("Restart Git Extensions");
```

Can we use `AppSettings.ApplicationName` instead here?

Reply:
> Can we use `AppSettings.ApplicationName` instead here?

I guess it is "GitExtensions" without the space (for downward compatibility of settings folder).